src/main/clojure/clara/rules/listener.cljc

```diff
@@ -23,6 +23,7 @@
   (remove-activations! [listener node activations])
   (fire-activation! [listener activation resulting-operations])
   (fire-rules! [listener node])
+  (activation-group-transition! [listener original-group new-group])
```

Comment:
I agree with the premise here, we've had breaking changes to listeners in the past and while not a big deal it still impacts consumers.
> This isn't an issue to be solved in this particular work.
It might be worth it to log an issue to look into ways that we can change listeners in the future to be less impactful. For now i think i am ok with adding a new method to this protocol, and announcing it as a breaking change, like we did in the past.

Reply:
I've logged https://github.com/cerner/clara-rules/issues/457 to look into a better approach to this with a thought on how to do it.  For now I think we can do as Ethan suggested above.